packages/asset-swapper/src/utils/market_operation_utils/sampler.ts

```diff
@@ -135,7 +135,9 @@ export class DexOrderSampler {
     public async executeBatchAsync<T extends Array<BatchedOperation<any>>>(ops: T): Promise<any[]> {
         const callDatas = ops.map(o => o.encodeCall(this._samplerContract));
         // Execute all non-empty calldatas.
-        const rawCallResults = await this._samplerContract.batchCall(callDatas.filter(cd => cd !== '0x')).callAsync();
+        const rawCallResults = await this._samplerContract
+            .batchCall(callDatas.filter(cd => cd !== '0x'))
+            .callAsync(this._callDataOverrides);
```

Comment:
Yeah I feel like it makes more sense for `_callDataOverrides` to be of `GethCallOverrides` and we do a `callAsync({ overrides: this._gethCallOverrides })` here.